flakehell/_cli.py

```diff
@@ -4,13 +4,18 @@
 
 # external
 from termcolor import colored
+import colorama
 
 # app
 from ._constants import ExitCode
 from ._types import CommandResult
 from .commands import COMMANDS
 
 
+# use colorama to make termcolor work on Windows
+colorama.init()
```

Comment:
Maybe no need to fix it on systems where it works already? 

```suggestion
if os.name == 'nt'
    colorama.init()

```

Reply:
"On other platforms, calling init() has no effect" as written in https://pypi.org/project/colorama/
That's why I did it like this. However, if this check is to be applied anyways, it shall check for **win32**, not nt.

Reply:
However, we can do this check, but then the import os is needed. I cannot edit your suggestion

Reply:
ok, I have added this check to the PR